app/javascript/components/StartEndModal/StartEndModal.jsx

```diff
@@ -38,37 +39,33 @@ const StartEndModal = ({ gameState, startEnd, errors }) => {
   const [selectedTeamId, setSelectedTeamId] = useState(first_team_id || '');
 
   const [open, setOpen] = useState(false);
+  const [teams] = useState(teams_with_players.map((team, index) => {
+      return ( <MenuItem key={index} value={team.team.id}>{team.team.team_name}</MenuItem> );
+    })
+  );
 
   const errorsExist = errors && errors.throw_order;
   const label = errorsExist ? "First Team required*" : "First Team*";
   
   useEffect(() => {
-    if (currentEnd > 0 && ends[currentEnd - 1].end.score_team1) {
+   
+    if (currentEnd > 0 && ends[currentEnd - 1].end.score_team1 >= 0) {
```

Comment:
good catch. This wouldn't have worked in cases where team2 scored and it looks like I had the logic backwards.